internal/dot/graph.go

```diff
@@ -92,6 +131,8 @@ type Graph struct {
 	Groups   []*Group
 	groupMap map[groupKey]*Group
 
+	paramCtorMap map[nodeKey][]*Ctor
```

Comment:
nit: Can call this "consumers" to replicate what dig's Container does.

I know this is already done in the existing code, but we should avoid naming
things after their type since they're strongly typed and we already know the
type.

You can name it after its purpose too: It's a reverse index to find
constructors given the parameter type, so you can name it "ctorParamIndex" as
well. "consumers" is equally informative because it says you're looking for
things that consume values of this type.

Reply:
That make sense, I think consumer is a good name!